app/models/assignment.rb

```diff
@@ -697,17 +691,6 @@ def next_criterion_position
     get_criteria.count > 0 ? get_criteria.last.position + 1 : 1
   end
 
-  # Returns the class of the criteria that belong to this assignment.
-  def criterion_class
```

Comment:
Thank you. I was thinking about using criterion_id, but I could get another criterion with a different type. I am trying to pass both the "criterion_id" and the "criterion_type" through the form.